### PR TITLE
cli: mkdir, commit message, clean-up

### PR DESCRIPTION
**Pull request by jirikuncar**
see commit messages

### CODE REVIEW

**Comment by jirikuncar** [comment]
@rokroskar can you please have a look so I can make a release today?

File: renku/cli/init.py [hunk 1]
```diff
@@ -36,6 +36,30 @@
 
 This creates a new subdirectory named ``.renku`` that contains all the
 necessary files for managing the project configuration.
+
+Updating an existing project
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are situations when the required structure of a Renku project needs
+to be recreated or you have an **existing** Git repository. You can solve
+these situation by simply adding ``--force`` option.
```
Comment (rokroskar) [inline renku/cli/init.py]: adding the

**Comment by rokroskar** [comment]
sorry, I didn't realize there was a new commit here. Looks good, thanks!

File: renku/cli/init.py (hunk 1 above)
Comment (jirikuncar) [reply renku/cli/init.py]: fixed in next PR

**Pull request merged**